### ISSUE TITLE
CPU环境下运行测试代码一直卡住是怎么回事？

### ISSUE THREAD

**Issue by chenmj201601** [FunAudioLLM/CosyVoice#678]
CPU环境，

`
from cosyvoice.cli.cosyvoice import CosyVoice
from cosyvoice.utils.file_utils import load_wav
import torchaudio
import torch

cosyvoice = CosyVoice('/workspace/models/iic/CosyVoice-300M-SFT')
print(cosyvoice.list_avaliable_spks())
# change stream=True for chunk stream inference
audio=[]
for i, j in enumerate(
    cosyvoice.inference_sft(
        '8月21日上午，市政府新闻办举行市政府新闻发布会，介绍临港新片区四周年发展成就及下一步总体考虑等有关情况。', 
        '中文女', 
        stream=False)):
    audio.append(j['tts_speech'])

torchaudio.save('sft.wav', torch.cat(audio, dim=1), 22050)
`

以上运行后就一直卡住，cpu占用很高。

不能在cpu环境推理吗，我cpu是i9的，主频有3.7G。

**Comment by aluminumbox** [comment]
可以cpu推理，可以加日志看看具体是哪里是不是采样死循环
